cli/cmd/brokers.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(brokersCmd)
```

Comment:
Yes, my understanding was that you should be able to do the following:
 - manage brokers [brokers.go]
 - manage service instances [services.go]
 - manage bindings [bindings.go]

Did I misunderstand?